packages/mdc-list/index.ts

```diff
@@ -33,8 +33,8 @@ class MDCList extends MDCComponent<MDCListFoundation> {
     this.foundation_.setVerticalOrientation(value);
   }
 
-  get listElements(): Element[] {
-    return [].slice.call(this.root_.querySelectorAll(strings.ENABLED_ITEMS_SELECTOR));
+  get listElements(): HTMLElement[] {
```

Comment:
I think it was because we're calling `.focus()` on these items in a few places, and I assumed `.mdc-list-item` would always be an `HTMLElement` anyway, so I thought we could avoid type assertions by just declaring it an `HTMLElement`.

However, it looks like it's pretty easy to change this back to `Element` if you prefer (take a look at commit 91f5975bfc68af5c21a30ad34653fee6a080683e).

I figured it's probably OK to _return_ a more specific subtype, as long as our _arguments_ are still `Element` for Closure users.

WDYT?